src/main/java/com/getindata/connectors/http/sink/HttpSink.java

```diff
@@ -0,0 +1,78 @@
+package com.getindata.connectors.http.sink;
+
+import org.apache.flink.connector.base.sink.AsyncSinkBase;
+import org.apache.flink.connector.base.sink.writer.BufferedRequestState;
+import org.apache.flink.connector.base.sink.writer.ElementConverter;
+import org.apache.flink.core.io.SimpleVersionedSerializer;
+import org.apache.flink.util.Preconditions;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Collections;
+
+public class HttpSink<InputT> extends AsyncSinkBase<InputT, HttpSinkRequestEntry> {
```

Comment:
Hi,
this is maybe for @maver1ck 
From Flink's `AsyncSinkBase` doc, we can read that `"We are not considering support for exactly-once semantics at this point."`

This means that client's REST API that we will use must be idempotent.

Also I'm wondering about recovery from checkpoint with this source. 
In this scenario we will trigger X REST calls (PUT probably) with same data that was used before cluster failure.

Something to think about in upcoming future.